src/components/AboutUsSection/style.ts

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  justify-items: center;
+  margin-top: 0;
+`;
+
+export const TextArea = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 825px;
+  text-align: center;
+  margin: 0;
+
+  @media screen and (max-width: 1668px) {
```

Comment:
wdyt about adding media query breakpoints to theme?
```
breakpoints: {
  mobile: 'XXXpx',
  tablet: 'XXXpx'
  desktop: 'XXXpx',
   // or
  xs: 'XXXpx'
  sm: 'XXXpx'
}
```
and then:
`  @media screen and (max-width: ${props => props.theme.breakpoints.xxx}) {`